docs-v2/reference/sdks/node.mdx

```diff
@@ -167,8 +167,8 @@ await nango.getIntegration(<INTEGRATION-ID>);
 **Parameters**
 
 <Expandable>
-    <ResponseField name="unique_key" type="string" required>
-         The integration ID (`unique_key`)
+    <ResponseField name="uniqueKey" type="string" required>
+         The integration ID (`unique_Key`)
```

Comment:
should be `uniqueKey` right?

Reply:
no it was to reference the actual name in the object since it's not camelcased but it's confusing I'll remove it